tests/virt_autotest/guest_installation_run.pm

```diff
@@ -86,6 +86,19 @@ sub run {
         $upload_guest_assets_flag = 'yes';
     }
 
+    if (check_var('KERNEL_64KB_PAGE_SIZE', '64kpg')) {
+        # Verify 64kb page size enabled.
+        record_info('Baremetal kernel cmdline', script_output('cat /proc/cmdline'));
+        assert_script_run("grep -- '-64kb' /proc/cmdline");
+        record_info('INFO', '64kb page size enabled.');
+
+        # Swap needs to be reinitiated
+        my $swap_partition = script_output("swapon | awk '/\\/dev/{print \$1; exit}'");
+        record_info('Current swap partition is ', $swap_partition);
+        assert_script_run("swapoff $swap_partition");
+        assert_script_run('swapon --fixpgsz');
+    }
+
```

Comment:
About the kernel checking and swap configuration in guest_installation_run.pm, it would be better to move it ahead in login_console.pm to aviod unnecessary install_package.pm and update_packages.pm if 64k kernel is not  running?

Reply:
I was going to give similar suggestion :)

Reply:
I was going to give similar suggestion ;)

Reply:
Done, moved it to login_console.pm